### PR TITLE
make axp192 led settings in ttgobeam10.h to control led behavior

### PR DESCRIPTION
**Pull request by hekopath**

### CODE REVIEW

File: src/power.cpp [hunk 1]
```diff
@@ -78,6 +78,9 @@ void AXP192_power(pmu_power_t powerlevel) {
 
   case pmu_power_sleep:
     pmu.setChgLEDMode(AXP20X_LED_OFF);
+    #ifdef PMU_LED_RUN_MODE
+    pmu.setChgLEDMode(PMU_LED_SLEEP_MODE);
+    #endif
```
Comment (cyberman54) [inline src/power.cpp:83]: Why not

```
#ifdef PMU_LED_RUN_MODE
pmu.setChgLEDMode(PMU_LED_SLEEP_MODE);
#else
pmu.setChgLEDMode(AXP20X_LED_OFF);
#endif
```

?

File: src/power.cpp [hunk 2]
```diff
@@ -90,6 +93,9 @@ void AXP192_power(pmu_power_t powerlevel) {
     pmu.setPowerOutPut(AXP192_DCDC2, AXP202_OFF); // unused on T-Beam v1.0
     pmu.setPowerOutPut(AXP192_EXTEN, AXP202_OFF); // unused on T-Beam v1.0
     pmu.setChgLEDMode(AXP20X_LED_LOW_LEVEL);
+    #ifdef PMU_LED_RUN_MODE
+    pmu.setChgLEDMode(PMU_LED_RUN_MODE);
```
Comment (cyberman54) [inline src/power.cpp:97]: Why not

```
#ifdef PMU_LED_RUN_MODE
pmu.setChgLEDMode(PMU_LED_RUN_MODE);
#else
pmu.setChgLEDMode(AXP20X_LED_LOW_LEVEL);
#endif
```
?

**Review by cyberman54 — CHANGES_REQUESTED** [review]
Thanks for your PR! We should add this feature. Please see comments in code.

File: src/power.cpp (hunk 1 above)
Comment (hekopath) [reply src/power.cpp:83]: Propably the more coherent solution.

File: src/power.cpp (hunk 2 above)
Comment (hekopath) [reply src/power.cpp:97]: Propably the more coherent solution.

**Comment by cyberman54** [comment]
Manually merged to development branch.

**Pull request closed without merging**